specify/specify7
# Scope tree def items in query builder

Why wasn't this done before? oh well

To test:
1. Use a database that has multiple trees for every tree table.
2. Make a query in the query builder using TREEdefitem as base table, and choose any field. Choose 'name' for testing.
3. Check that production gives all the tree ranks. For example, you will find "Life" rank twice in production. Verify that only tree ranks in your scope will be visible.
For example, first is production, second is this branch
![image](https://github.com/specify/specify7/assets/61122018/5b68d960-4c34-451a-9fc3-36542d6b9394)
![image](https://github.com/specify/specify7/assets/61122018/0281ee19-0967-4666-8756-6709487c475b)

Note:
When testing in context of statistics, remove the taxa represented category and the 'error' stats. You shouldn't be able to add those items back when going to Add Stats Dialog
